database/executiveQuery.go

```diff
@@ -0,0 +1,32 @@
+package database
+
+import (
+	"context"
+
+	"github.com/UBC-Product-Management-Club/website-server/models"
+	"google.golang.org/api/iterator"
+)
+
+// Add an executive to database
+func AddExecutive(exec models.Executive) error {
+	_, _, err := client.Collection("executive").Add(context.Background(), exec)
```

Comment:
just wondering, what are the three variables returned by this operation? To my knowledge it's just error and document id? just curious

Reply:
Yeah, you are correct. It returns a reference to the document (Id), write result and error. Write result and error somewhat overlaps with each other because if error is `nil` then we know that the write is successful.